buildSrc/src/main/java/org/elasticsearch/gradle/JdkDownloadPlugin.java

```diff
@@ -30,6 +33,7 @@
     private static final String REPO_NAME_PREFIX = "jdk_repo_";
     private static final String EXTENSION_NAME = "jdks";
     public static final String JDK_TRIMMED_PREFIX = "(jdk-?\\d.*)|(zulu-?\\d.+).jdk";
+    public static final String ZULU_LINUX_AARCH_PATTERN = "zulu.*linux_aarch64";
```

Comment:
distress with this root folder pattern are packaged differently and we're going to ignore the trimmed prefix pattern